src/axom/inlet/Field.hpp

```diff
@@ -210,7 +210,7 @@ class Field : public std::enable_shared_from_this<Field>
    * \return Shared pointer to this Field instance
    *****************************************************************************
   */
-  std::shared_ptr<Field> validValues(std::vector<std::string> set);
+  std::shared_ptr<Field> validStringValues(std::vector<std::string> set);
```

Comment:
I know i told you to do this.. but the overloaded parameter is enough info.  you can drop the "String" from the parameter name.